tour/beerDEcoded.yaml

```diff
@@ -0,0 +1,188 @@
+    content: 'Now that your data is ready, let''s use some tools.<br>'
+    backdrop: true
+  - title: <b>Assign taxonomic classifications</b>
+    content: >-
+      One of the key steps in metagenomic data analysis is to identify the taxon
+      to which the individual reads belong. Taxonomic classification tools are
+      based on microbial genome databases to identify the origin of each
+      sequence.<br>
+    backdrop: true
+  - title: <b>Taxonomic classification with Kraken2</b>
+    content: >-
+      To perform the taxonomic classification we will use Kraken2. This tool
+      uses the minimizer method to sample the k-mers (all the read’s
+      subsequences of length k) in a deterministic fashion in order to reduce
+      memory constumption and processing time. In addition, it masks
+      low-complexity sequences from reference sequences by using dustmasker.<br>
+    backdrop: true
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: .toolMenuContainer
+    intro: Available tools appear here in the <b>tool menu</b>.
+    position: right
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: '#__BVID__106'
+    content: >-
+      You can use <b>'tool search'</b> to locate tools.<br><br> Search for 
+      <b>'Kraken2'</b> and select it.<br><br> Tools may take a couple of moments
+      to load.
+    placement: right
+  - title: <b>Taxonomic classification with Kraken2</b>
+    element: .toolMenuContainer
+    intro: >-
+      Open <b>'Kraken2'</b> tool<br><br> Click <b>'Next'</b> to continue our
+      tour.
+    position: right
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    element: 'div[tour_id="use_names"]'
+    intro: Have a look at the tool's parameters of the <b>'Kraken2'</b> tool.
+    placement: right
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    intro: >-
+      Please select the following parameters:<br><br> <b>'Single or paired
+      reads'</b>: Single <br> <b>'Input sequences'</b>: Uploaded dataset<br>
+      <b>'Print scientific names instead of just taxids'</b>: Yes<br>
+      <b>'Confidence'</b>: 0.0<br> <b>'In “Create Report”'</b>:<br>
+      &nbsp;&nbsp;&nbsp;&nbsp;<b>'Print a report with aggregrate counts/clade to
+      file'</b>: Yes<br> &nbsp;&nbsp;&nbsp;&nbsp;<b>'Format report output like
+      Kraken 1’s kraken-mpa-report'</b>: Yes<br> <b>'Select a Kraken2
+      database'</b>: fungi2019-03' <br><br>Click <b>'Next'</b> and the tour will
+      <b>'Execute'</b> the Kraken2 tool for you.'
+    postclick:
+      - '#execute'
+  - title: <b>Assign taxonomic labels with Kraken2</b>
+    element: '#current-history-panel > div.controls'
+    intro: >-
+      Congratulations, you have created two files. It contains Classification
+      and Report file.<br><br> It will remain stored in your history.<br><br>
+      Click the <b>'eye'</b> icon to view the data once the history item turns
+      green.
+    position: left
```

Comment:
I propose to add one additional step with additonal information about Classification file anf Report file format of Kraken2:
"<small>Additional info:<br>
      In <b>classification file</b> each sequence (or sequence pair, in the case of paired reads) classified by Kraken 2 results in a single line of output. Kraken 2's output lines contain five tab-delimited fields; from left to right, they are:
      <ol><li>"C"/"U": a one letter code indicating that the sequence was either classified or unclassified.</li><li>The sequence ID, obtained from the FASTA/FASTQ header.</li><li>The taxonomy ID Kraken 2 used to label the sequence; this is 0 if the sequence is unclassified.</li><li>The length of the sequence in bp. In the case of paired read data, this will be a string containing the lengths of the two sequences in bp, separated by a pipe character, e.g. "98|94".</li><li>A space-delimited list indicating the LCA mapping of each k-mer in the sequence(s). For example, "562:13 561:4 A:31 0:1 562:3" would indicate that:<br><ul><li>the first 13 k-mers mapped to taxonomy ID #562</li><li>the next 4 k-mers mapped to taxonomy ID #561</li><li>the next 31 k-mers contained an ambiguous nucleotide</li><li>the next k-mer was not in the database</li><li>the last 3 k-mers mapped to taxonomy ID #562</li></ul>Note that paired read data will contain a "|:|" token in this list to indicate the end of one read and the beginning of another.When Kraken 2 is run against a protein database (see [Translated Search]), the LCA hitlist will contain the results of querying all six frames of each sequence. Reading frame data is separated by a "-:-" token.</li></ol>
      Kraken 2's standard sample <b>report file</b> format is tab-delimited with one line per taxon. The fields of the output, from left-to-right, are as follows:<ol><li>Percentage of fragments covered by the clade rooted at this taxon</li><li>Number of fragments covered by the clade rooted at this taxon</li><li>Number of fragments assigned directly to this taxon</li><li>A rank code, indicating (U)nclassified, (R)oot, (D)omain, (K)ingdom, (P)hylum, (C)lass, (O)rder, (F)amily, (G)enus, or (S)pecies. Taxa that are not at any of these 10 ranks have a rank code that is formed by using the rank code of the closest ancestor rank with a number indicating the distance from that rank. E.g., "G2" is a rank code indicating a taxon is between genus and species and the grandparent taxon is at the genus rank.</li><li>NCBI taxonomic ID number</li><li>Indented scientific name</li></ol>The scientific names are indented using space, according to the tree structure specified by the taxonomy.</small>
"

Reply:
Would be good to also link your source. (Kraken manual)

Reply:
done